### ISSUE TITLE
Landscape optimization

### ISSUE THREAD

**Issue by marcosci** [ropensci/landscapetools#20]
Would be nice to have the functionality of LG (http://www.lg.ethz.ch/).
Would also mean that we need landscapemetrics and NLMR as dependencies, as well as the GenAlg package.

Any opinions? Or another package for that?

**Comment by bitbacchus** [comment]
I think it would make sense to have it in NLMR (to generate neutral landscapes that have a certain configuration/composition).

**Comment by mhesselbarth** [comment]
I worked on something similar for rasters, so maybe we wouldn't need the GenAlg dependency

**Comment by marcosci** [comment]
True, for shar, or? But here or in another package?
There were 2 recent publications using LG (but otherwise R) ... would be nice to provide the same functionality in R.

**Comment by mhesselbarth** [comment]
It was in one of the early early versions of shar, yes. I ditched it at some point, but I'll try to find the old code. Would definitely need some modifications, but maybe nice as a starting point

**Comment by Nowosad** [comment]
Good idea. I used it for one project about a year ago, but finally I decided on using NLMR. I even wrote an R wrapper - https://gitlab.com/nowosad/landscapegenerator. 

I think it should be either (1) a new package or (2) a part of NLMR.
